scss/window/_theme.scss

```diff
@@ -4,6 +4,9 @@
     $window-shadow: 0 3px 3px 0 rgba(0, 0, 0, .06) !default;
     $window-focused-shadow: 1px 1px 7px 1px rgba(0, 0, 0, .3) !default;
 
+    $window-titlebar-border-color: inherit !default;
+    $window-titlebar-bg: $accent none !default;
+    $window-titlebar-text: $accent-contrast !default;
```

Comment:
Those should be tied to `$header-` variables, not `$accent`. You can customize them later